docs/src/main/sphinx/connector/phoenix.rst

```diff
@@ -94,40 +94,120 @@ that catalog name instead of ``phoenix`` in the above examples.
 Type mapping
 ------------
 
-The data type mappings are as follows:
-
-==========================   ============
-Phoenix                      Trino
-==========================   ============
-``BOOLEAN``                  (same)
-``TINYINT``                  (same)
-``UNSIGNED_TINYINT``         ``TINYINT``
-``SMALLINT``                 (same)
-``UNSIGNED_SMALLINT``        ``SMALLINT``
-``INTEGER``                  (same)
-``UNSIGNED_INTEGER``         ``INTEGER``
-``BIGINT``                   (same)
-``UNSIGNED_LONG``            ``BIGINT``
-``FLOAT``                    ``REAL``
-``UNSIGNED_FLOAT``           ``FLOAT``
-``DOUBLE``                   (same)
-``UNSIGNED_DOUBLE``          ``DOUBLE``
-``DECIMAL``                  (same)
-``BINARY``                   ``VARBINARY``
-``VARBINARY``                (same)
-``TIME``                     (same)
-``UNSIGNED_TIME``            ``TIME``
-``DATE``                     (same)
-``UNSIGNED_DATE``            ``DATE``
-``CHAR``                     (same)
-``VARCHAR``                  (same)
-``ARRAY``                    (same)
-==========================   ============
-
-The Phoenix fixed length ``BINARY`` data type is mapped to the Trino
-variable length ``VARBINARY`` data type. There is no way to create a
-Phoenix table in Trino that uses the ``BINARY`` data type, as Trino
-does not have an equivalent type.
+Because Trino and Phoenix each support types that the other does not, this
+connector modifies some types when reading or writing data. Data types may not
+map the same way in both directions between Trino and the data source. Refer to
+the following sections for type mapping in each direction.
+
+Phoenix type to Trino type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps Phoenix types to the corresponding Trino types following this
+table:
+
+.. list-table:: Phoenix type to Trino type mapping
+  :widths: 30, 20
+  :header-rows: 1
+
+  * - Phoenix database type
+    - Trino type
+  * - ``BOOLEAN``
+    - ``BOOLEAN``
+  * - ``TINYINT``
+    - ``TINYINT``
+  * - ``UNSIGNED_TINYINT``
+    - ``TINYINT``
+  * - ``SMALLINT``
+    - ``SMALLINT``
+  * - ``UNSIGNED_SMALLINT``
+    - ``SMALLINT``
+  * - ``INTEGER``
+    - ``INTEGER``
+  * - ``UNSIGNED_INTEGER``
+    - ``INTEGER``
+  * - ``BIGINT``
+    - ``BIGINT``
+  * - ``UNSIGNED_LONG``
+    - ``BIGINT``
+  * - ``FLOAT``
+    - ``REAL``
+  * - ``UNSIGNED_FLOAT``
+    - ``FLOAT``
+  * - ``DOUBLE``
+    - ``DOUBLE``
+  * - ``UNSIGNED_DOUBLE``
+    - ``DOUBLE``
+  * - ``DECIMAL``
+    - ``DECIMAL``
+  * - ``BINARY``
+    - ``VARBINARY``
+  * - ``VARBINARY``
+    - ``VARBINARY``
+  * - ``TIME``
+    - ``TIME``
+  * - ``UNSIGNED_TIME``
+    - ``TIME``
```

Comment:
`TIME` and `UNSIGNED_TIME` are unsupported. Let's remove these entries.

Reply:
`TIME ` still exists. Please remove it from read mapping.

Reply:
Could you put `TIME` only in write mapping? It's little weird, but this's the current status.